String_Programs/Vowel_String.py

```diff
@@ -0,0 +1,12 @@
+String = input('Enter the string : \n')
+count = 0
+String = String.lower()
+for i in String:
+    if i == 'a' or i == 'e' or i == 'i' or i == 'o' or i == 'u':
```

Comment:
Use in operator: https://realpython.com/python-in-operator. Don't compare one value at a time.